Elastic Serverless Forwarder support for Logstash (#2500)
elastic/observability-docs: docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

`inputs.[].outputs.[].type`:

The type of the forwarding target output. Currently only the following outputs are supported:

 * `elasticsearch`

>  * `logstash`

Reviewer: ```suggestion
 * preview:[] `logstash`
```
I think you might have applied this before I edited it to include the preview admonition.

`inputs.[].outputs.[].type`:

> The type of the forwarding target output (currently only `elasticsearch` and `logstash`preview:[Technical Preview] are supported).

Reviewer: ```suggestion
The type of the forwarding target output. Currently only the following outputs are supported:

 * `elasticsearch`
 * `logstash`
```

`inputs.[].outputs.[].type`:

The type of the forwarding target output (currently only `elasticsearch` and `logstash` are supported).

> NOTE: `logstash` target output is currently offered as Technology Preview.**

Reviewer: Do you mean to stay technical preview here? Here is the boiler plate text that gets added when you use the [technical preview admonition](https://github.com/elastic/docs/blob/master/README.asciidoc#using-the-technical-preview-admonition) in the docs: 

![image](https://user-images.githubusercontent.com/14206422/216503365-4bbf8340-22b0-4249-9cef-553c5645b6e9.png)

Is technology preview different of the same? If the same, we should add text that matches.

> NOTE: `logstash` target output is currently offered as Technology Preview.**

Reviewer: ```suggestion
NOTE: The `logstash` target output is currently offered as a **Technology Preview.**
```
Editing in case you keep this. Looks like you wanted this to be bold?

  * `args.batch_max_bytes`: (Optional) Maximum size in bytes to send in a single bulk request. Default value: 10485760 (10MB).
  * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport.

For `logstash` the following arguments are supported:

>   * `args.logstash_url` URL of elasticsearch endpoint in the format `http(s)://host:port`

Reviewer: ```suggestion
  * `args.logstash_url`: URL of {ls} endpoint in the format `http(s)://host:port`
```

Reviewer: Shouldn't this say Logstash endpoint? I'm using the shared attribute [defined in this file](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L239) to resolve the product name. (Of course we can use lowercase `logstash` when referring to the target output type.)

>   * `args.username`: (Optional) Username of the logstash instance to connect to. Mandatory if HTTP Basic authentication is enabled in Logstash.

Reviewer: ```suggestion
  * `args.username`: (Optional) Username of the {ls} instance to connect to. Mandatory if HTTP Basic authentication is enabled in {ls}.
```

>   * `args.password`: (Optional) Password of the logstash instance to connect to. Mandatory if HTTP Basic authentication is enabled in Logstash.

Reviewer: ```suggestion
  * `args.password`: (Optional) Password of the {ls} instance to connect to. Mandatory if HTTP Basic authentication is enabled in {ls}.
```

  * `args.max_batch_size`: (Optional) Maximum number of events to send in a single HTTP(s) request. Default value: 500

>   * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards logstash. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1

Reviewer: ```suggestion
  * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards {ls}. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1
```
